doc/README.hpux

```diff
@@ -155,7 +155,7 @@ promiscuous mode support.
 
 An additional note, from Jost Martin, for HP-UX 10.20:
 
-	Q: How do I get ethereral on HPUX to capture the _outgoing_ packets
+	Q: How do I get [ethereal] on HPUX to capture the _outgoing_ packets
```

Comment:
This is a brand. Since this is a quote from a post, I'm using brackets to correct the spelling.

Reply:
Well, at the time, it was the thing now known as "wireshark".  If it's worth keeping this README.hpux, then maybe it's worth changing this to "wireshark"

Reply:
(With "Wireshark", capitalized, in brackets, to indicate that this is a change from the original version of the quote.)